src/components/BounceRuleDetails/index.jsx

```diff
@@ -72,3 +73,11 @@ export default class BounceRuleDetails extends React.Component {
     );
   }
 }
+
+BounceRuleDetails.propTypes = {
```

Comment:
Nice let's continue to add proptypes as we go to each of the stateful class components